oneanddone/urls.py

```diff
@@ -24,16 +22,15 @@ def handler404(request):
     return render(request, '404.html', status=404)
 
 
-urlpatterns = patterns('',
+urlpatterns = patterns(
+    '',
```

Comment:
Since 1.8 or 1.7 there is a new style for urlpatterns https://docs.djangoproject.com/en/1.8/topics/http/urls/#example

Also I thought that I had that fixed for sugardough, do you mind r? @bobsilverberg https://github.com/mozilla/sugardough/pull/97